scrabble.js

```diff
@@ -1,15 +1,154 @@
+const scoreTable = {
+  'a': { 'points':  1, 'tiles':  9 },
+  'b': { 'points':  3, 'tiles':  2 },
+  'c': { 'points':  3, 'tiles':  2 },
```

Comment:
I like that you've consolidated the points and number of tiles into one data structure - clever thinking!